test/e2e/deployment.go

```diff
@@ -443,27 +444,42 @@ func (d *Deployment) EnsureResourcesForLocalContour() error {
 	}
 
 	// Add bootstrap ConfigMap as volume on Envoy pods (also removes cert volume).
```

Comment:
nit: add comment on the admin volume as well

Reply:
👍